common/src/main/java/com/microsoft/identity/common/crypto/AndroidWrappedKeyLoader.java

```diff
@@ -87,22 +85,26 @@ public class AndroidWrappedKeyLoader extends AES256KeyLoader {
     private static final String WRAP_KEY_ALGORITHM = "RSA";
 
     /**
-     * Indicate that token item is encrypted with the key persisted in AndroidKeyStore.
+     * Indicate that token item is encrypted with the key loaded in this class.
      */
-    public static final String KEY_IDENTIFIER = "A001";
-
-    /**
-     * Name of the file contains the symmetric key used for encryption/decryption.
-     */
-    /* package */ static final String KEY_FILE_PATH = "adalks";
+    public static final String WRAPPED_KEY_KEY_IDENTIFIER = "A001";
```

Comment:
Since the other one is named as USER_PROVIDED_KEY_IDENTIFIER, should we name this as 
AUTH_SDK_PROVIDED_KEY_IDENTIFIER?

Reply:
Broker's going to use this as well.

Reply:
````suggestion
    public static final String AUTH_LIB_PROVIDED_KEY_IDENTIFIER = "A001";
````
or maybe this could be
````suggestion
    public static final String DEFAULT_KEY_IDENTIFIER = "A001";
````
and the other one can be APPLICATION_PROVIDED_KEY_IDENTIFIER

Feel free to won't fix this.

Reply:
A001 is going to be used for both broker and auth libs though (and it's not configurable as of now).

i'll mark this as won't fix.